### PR TITLE
ci: Fix Docs CI Workflow

### PR DESCRIPTION
**Pull request by isaac-chung**
Addresses https://github.com/embeddings-benchmark/mteb/issues/620

CI was failing with:
```
Run python docs/create_tasks_table.py
Traceback (most recent call last):
  File "/home/runner/work/mteb/mteb/docs/create_tasks_table.py", line 85, in <module>
    main()
  File "/home/runner/work/mteb/mteb/docs/create_tasks_table.py", line 81, in main
    insert_table(file_path, table)
  File "/home/runner/work/mteb/mteb/docs/create_tasks_table.py", line 67, in insert_table
    md = md.replace(md[md.index(start) + len(start) : md.index(end)], table)
ValueError: substring not found
Error: Process completed with exit code 1.
```
which references the code here:
```python
start = "<!-- TABLE START -->"
end = "<!-- TABLE END -->"

md = md.replace(md[md.index(start) + len(start) : md.index(end)], table)
```

### CODE REVIEW

**Comment by isaac-chung** [comment]
Maybe @KennethEnevoldsen or @imenelydiaker , whoever gets to this first.

**Comment by isaac-chung** [comment]
Maybe @KennethEnevoldsen or @imenelydiaker , whoever gets to this first.

**Review by imenelydiaker — APPROVED** [review]
LGTM! Thanks for fixing this, you can merge it 🙂

File: docs/mmteb/points/621.jsonl [hunk 1]
```diff
@@ -0,0 +1,2 @@
+{"GitHub": "isaac-chung", "Bug fixes": 2}
+{"GitHub": "???", "Review PR": 2}
```
Comment (isaac-chung) [inline docs/mmteb/points/621.jsonl:2]:
```suggestion
{"GitHub": "imenelydiaker", "Review PR": 2}
```

**Comment by KennethEnevoldsen** [comment]
Thanks for fixing this @isaac-chung!

**Pull request merged**